src/workflow/snakemake/scripts/run_module.py

```diff
@@ -4,52 +4,147 @@
 ##
 ## Started 22 Feb 2024
 ## Izaskun Mallona
```

Comment:
I think is time to remove these dirty comments about old authors and attribute (all/actual) contributors properly